routing_packager_app/routers/router_base.py

```diff
@@ -36,7 +39,9 @@ def __init__(self, provider, input_pbf_path):
                                            )
         volumes = {host_dir: {'bind': '/app/data', 'mode': 'rw'}}
         try:
-            self._container = docker_clnt.containers.create(self.image, volumes=volumes)
+            self._container: Optional[Container] = docker_clnt.containers.create(
+                self.image, volumes=volumes, command="tail -f /dev/null"
+            )
```

Comment:
problem was that we updated our valhalla docker image in the meantime and it would simply exit if the server doesn't find valhalla tiles. need to add custom command to make it run smoothly before we build tiles inside.